src/mesh/python/x3d_generator.py

```diff
@@ -117,13 +124,14 @@
 for cell in range(mesh.num_cells):
     for j in range(mesh.num_faces_per_cell[cell]):
         # -- get the full face index for the cell and local face index
-        face = mesh.faces_per_cell[cell, j]
+        #face = mesh.faces_per_cell[cell, j]
```

Comment:
What is the motivation for changing the numpy array dimensions here (it looks equivalent, but I am probably missing something)?

Reply:
This is to allow for `faces_per_cell` to be a list of arrays rather than a 2D array -- for the voronoi mesh I can have any number of faces per cell so the 2D array doesn't work

Reply:
Oh right - I guess I accidentally baked in orth_mesh assumptions in this file.